modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -550,6 +550,11 @@ const ActionForm = ({
 									onChange={(event) =>
 										setActionData({
 											...actionData,
+											method:
+												event.target.value ===
+												ACTION_TYPE.ASYNC
+													? ACTION_METHOD.DELETE
+													: '',
```

Comment:
I'm ok with being opinionated with default, but it seems unnecessarily sneaky to add it in handler. Could we instead set it in UI? Something like
```js
value={actionData.method || ACTION_METHOD.DELETE}
```
on [this line](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L592). The field is required, so it cannot be unset by user, so it should end up applied [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L227-L229).

Reply:
Yep, much better :+1: